<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-SFOUNT5" ignored="0" name="Test" tests="6" failures="1" timestamp="09 Dec 2021 11:12:20 GMT" time="0.162" errors="0">
  <testcase name="tc1" time="0.038" classname="org.sample.Testing"/>
  <testcase name="tc2" time="0.005" classname="org.sample.Testing"/>
  <testcase name="tc3" time="0.006" classname="org.sample.Testing">
    <skipped/>
  </testcase> <!-- tc3 -->
  <testcase name="tc3" time="0.002" classname="org.sample.Testing">
    <skipped/>
  </testcase> <!-- tc3 -->
  <testcase name="tc3" time="0.002" classname="org.sample.Testing">
    <skipped/>
  </testcase> <!-- tc3 -->
  <testcase name="tc3" time="0.004" classname="org.sample.Testing">
    <failure type="java.lang.AssertionError" message="the test case failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: the test case failed expected [true] but found [false]
at org.sample.Testing.tc3(Testing.java:24)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- tc3 -->
</testsuite> <!-- Test -->
